### PR TITLE
CLI: add --version option

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

It's often useful to be able to quickly check version number of installed Ralph instance.

## Proposal

- [x] add `--version` CLI option

During this work, I also improved the following:

- [x] pydoctyle: ignore development folders
- [x] cli: homogenize commands help

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -119,7 +119,7 @@ exclude =
 
 [pydocstyle]
 convention = google
-match_dir = ^(?!tests).*
+match_dir = ^(?!tests|venv|build|scripts).*
```
Comment (quitterie-lcs) [inline setup.cfg:122]: Why did you add this ? I don't get clearly the point

Comment (jmaupetit) [reply setup.cfg:122]: I don't want to spend time linting those local directories only used for development and containing third-party sources.

**Review by wilbrdt — APPROVED** [review]
Looks good to me! 🎉
